src/pages/student/practice.js

```diff
@@ -85,6 +85,22 @@ const Practice = () => {
 	const { status, startRecording, stopRecording, mediaBlobUrl, clearBlobUrl } =
 		useReactMediaRecorder({ audio: true });
 
+	/* TODO: timer counting up feature to time practice */
+	const [practiceTime, setPracticeTime] = useState({ms:0, s:0, m:0, h:0});
+	const [time, setTime] = useState(0);
+	var updatedMs = practiceTime.ms, updatedS = practiceTime.s, updatedM = practiceTime.m, updatedH = practiceTime.h;
+
+	useEffect(() => {
+		const intervalId = setInterval(() => {
```

Comment:
Conversion from 60s to 1m and 60m to 1hr could take place here.

Reply:
could reset timer after new word is selected here